css/style.css

```diff
@@ -0,0 +1,67 @@
+body {
+	width: 100%;
+	width:1024px;
+	margin: auto;
+	background: url(images/fon3.jpg ) ;
+	background-repeat:  no-repeat; 
+	background-position: top center;
+	background-attachment: fixed;
+	background-size: cover ;
+	}
+
+.selection-title{
+	color: #FFA500;
+	}	
+h3 {
+	color: Maroon;
+}
+
+footer{
+	
+		flex: 0 0 auto;
+		height: 145px;
+		background: #3498DB;
+	
+}
+footer>ul
+{
```

Comment:
You have a lot of inconsistencies in the style of the code here and in the whole file. Such as:
1) For spaces within the rules, two spaces are used.
2) After the colon in the rules put one space.
3) Each declaration in the rule is written on a new line.
4) A single space is placed before the opening brace. After the bracket, the record is from a new line.
5) The closing brace is written on a new line and without an indent. The next rule after this is separated by an empty line.
Read more in http://codeguide.academy/html-css.html